### DISCUSSION TITLE
Unified_ESP8285_900_LR1121_RX_via_UART

### DISCUSSION THREAD

**Discussion by nexus5x11** [ExpressLRS/ExpressLRS#2985] [Q&A]
How can i build this and flash to my ELRS i bought from ALIBABA user these chips? no suport anywere

**Comment by JyeSmith** [comment]
ELRS isnt support the 8285 for new hardware, receivers and backpacks.  Espressif marked it EoL in 2 years.  So all LR1121 hardware will be ESP32 C3 based.

Out of interest do you have a link for the hardware?

**Reply by nexus5x11** [reply]
ah thanks extreme nightmare to find this, i did find a solfware on some Ukrainian telegram channel but it didnt work.
yea here is the link https://www.aliexpress.us/item/3256807355081726.html

**Comment by Z-Vladus** [comment]
Hello!
I have also managed to get RX wit ESP8285 + LR1121 on board.
The heading says: 
_CYCLONE SubGHz RX
Firmware Rev. 3.4.3 (SubGHz) FCC915_
Is there any chance for inexperienced user to compile the firmware ?
Getting some help on this will be really appreciated.

**Reply by pkendall64** [reply]
Cyclone receivers are unsupported as they do not participate in the project. Their receivers are untested by the development team.
The dev team also have no target for ESP8285 + LR1121, it was decided that we should push manufacturers towards the ESP32C3 as that is a much better platform going forward for small receivers.

So Cyclone really have shat on users by doing this.

**Reply by JyeSmith** [reply]
Can you please share an image of the web UI with the target name.

**Reply by Z-Vladus** [reply]
Sure. 
![2024-10-20_09-36-52](https://github.com/user-attachments/assets/95f9f2c5-89bb-4a9d-8897-556dd0263660)
![Clipboard_10-20-2024_01](https://github.com/user-attachments/assets/1a04f571-06ed-4725-8de3-53ab1aee93a8)

**Reply by JyeSmith** [reply]
@Z-Vladus  Can I trouble you for a better image of both sides?

**Reply by Z-Vladus** [reply]
Yeah,

![photo_2024-10-20_10-27-48](https://github.com/user-attachments/assets/b2b82e38-c1c9-45f5-a308-e3acb8daa868)
![photo_2024-10-20_10-28-23](https://github.com/user-attachments/assets/2ae52ef0-7685-40a1-8754-0aa93d5358bd)
